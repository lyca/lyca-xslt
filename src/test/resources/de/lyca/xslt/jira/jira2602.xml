<?xml version='1.0' encoding='utf-8' standalone='yes' ?>
<root>
  <item name="Item 1">
    <item name="Item 2">
      <item name="Item 3">
        <item name="Item 4" />
      </item>
    </item>
  </item>
</root>