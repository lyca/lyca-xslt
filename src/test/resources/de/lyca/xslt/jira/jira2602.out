<p>ancestor::item<ul>
<li>1 / 3 : Item 1</li>
<li>2 / 3 : Item 2</li>
<li>3 / 3 : Item 3</li>
</ul>
</p><p>ancestor::item/@name<ul>
<li>1 / 3 : Item 1</li>
<li>2 / 3 : Item 2</li>
<li>3 / 3 : Item 3</li>
</ul>
</p>
